playbooks/rfcs.md

```diff
@@ -39,6 +39,19 @@ Find the right repo:
 - Does it affect the whole dev team? (use [README](https://github.com/artsy/README))
 - Can't figure it out or needs to be private? (use [Potential][] as a fallback)
 
+Create a plan for how the RFC will be implemented. If the change is complex or large in scope consider defining a
+feedback process to iterate on the proposal as you implement and creating milestones and/or metrics to track
+progress.
+
+Decide how you'll resolve the RFC:
+
+- Specify whose feedback is necessary on this RFC with review requests.
+- Specify for how long feedback will be collected and incorporated into the proposal. At least a week for feedback
+  is appropriate for any proposal impacting all of engineering.
+- Specify how approval will ultimately be decided. Does this depend on X% of the engineering team actively
```

Comment:
Not sure what the exact wording should be here but the goal of an RFC is not necessarily to build consensus. We need to make sure that the barrier to creating and pushing RFC is low, and in most cases an RFC with no feedback and no dissent that is "safe to try" should be good to go. 

So maybe the default approval criteria is: "Lack of dissent / No objection" ?

Reply:
That's a great suggestion, to have a default, low barrier approval model as part of this. I'll update the PR.